big_tests/tests/domain_removal_SUITE.erl

```diff
@@ -197,6 +208,48 @@ private_removal(Config) ->
         ?assert_equal_extra(<<>>, Val2, #{stanza => Res2})
       end).
 
+roster_removal(Config) ->
+    escalus:fresh_story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
+        %% add contact
+        Stanza = escalus_stanza:roster_add_contact(Bob, [<<"friends">>], <<"Bobby">>),
+        escalus:send(Alice, Stanza),
+        Received = escalus:wait_for_stanzas(Alice, 2),
+        escalus:assert_many([is_roster_set, is_iq_result], Received),
+
+        %% check roster
+        BobJid = escalus_client:short_jid(Bob),
+        Received2 = escalus:send_iq_and_wait_for_result(Alice, escalus_stanza:roster_get()),
+        escalus:assert(is_roster_result, Received2),
+        escalus:assert(roster_contains, [BobJid], Received2),
+        escalus:assert(count_roster_items, [1], Received2),
+
+        {selected, [_]} = select_rosterusers(host_type(), domain()),
+        {selected, [_]} = select_rostergroups(host_type(), domain()),
+        {selected, [_]} = select_roster_version(host_type(), domain()),
+
+        %% remove domain and check roster
+        run_remove_domain(),
+        Received3 = escalus:send_iq_and_wait_for_result(Alice, escalus_stanza:roster_get()),
+        escalus:assert(is_roster_result, Received3),
+        escalus:assert(count_roster_items, [0], Received3),
+
+        {selected, []} = select_rosterusers(host_type(), domain()),
+        {selected, []} = select_rostergroups(host_type(), domain()),
+        {selected, []} = select_roster_version(host_type(), domain())
+        end).
+
+select_rosterusers(HostType, Domain) ->
```

Comment:
I suppose this function can be simplified like(name can be used other):

```erlang
select_from_roster(Table) ->
    Query = "SELECT * FROM " ++ Table ++ " WHERE server='" ++ binary_to_list(domain()) ++ "'",
    {selected, Res} = rpc(mim(), mongoose_rdbms, sql_query, [host_type(), Query]),
    Res.
```
using:
```erlang
[] = select_from_roster("rosterusers")
[_] = select_from_roster("rostergroups")
[] = select_from_roster("roster_version")
```